docs/architecture/server/index.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 30
+---
+
+# Server Architecture
+
+## CQRS ([ADR-0008](../adr/0008-server-CQRS-pattern.md))
+
+We are currently transitioning the server to use a Command and Query Responsibility Segregation
```

Comment:
ℹ️ I'd like to state things in a more present tense, as this is _what is_ happening per the ADR. ADRs imply change but when referred to as a part of current architecture they "are".

Reply:
Rephrased to remove the reference to the transition